pkgs/os-specific/linux/ryzen-monitor-ng/default.nix

```
@@ -0,0 +1,45 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "ryzen_monitor_ng";
```

```suggestion
  pname = "ryzen-monitor-ng";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md?plain=1#L357

---

Your proposed change conflicts with this rule:

https://github.com/NixOS/nixpkgs/blob/535caea95c5957391e878fe43fdaf81e0946e869/pkgs/README.md?plain=1#L347

And the name of the upstream package uses `_`

---

My apology

---

I will leave it as is.